emotioncf/cf.py

```diff
@@ -174,12 +174,15 @@ def plot_predictions(self):
 			f, ax = plt.subplots(nrows=1,ncols=3, figsize=(15,8))
 		else:
 			f, ax = plt.subplots(nrows=1,ncols=2, figsize=(15,8))
-
-		sns.heatmap(self.ratings,vmax=100,vmin=0,ax=ax[0],square=False,xticklabels=False, yticklabels=False)
+		
+		vmax = self.ratings.max().max() if self.ratings.max().max() > self.predicted_ratings.max().max() else self.predicted_ratings.max().max()
```

Comment:
This is fine, but we probably should have an option to pass in a vmin=None, vmax=None in the function.  If it is None, then default to autoscaling.  Sometimes you will want to plot figures that have a certain scaling.  You could also pass in a kwargs that will be input into the heatmap function.

Reply:
@ljchang Added kwargs support in dff9099